configs/setup_golden_image.py

```diff
@@ -41,17 +46,17 @@ def customize_golden(manconfig, golden_tar, build_dir):
         'golden_tar' : golden_tar,
         'build_dir' : build_dir,
         'status_file' : build_dir + '/status.json',
-        'verbose' : True,
+        'verbose' : VERBOSE,
```

Comment:
The librarian and other projects use a number to increase the output.  0 == no output, 1 == a little, 2 == more.....they get mapped to logger levels.

Reply:
When we were implementing logger, we agreed on using bool.
We can review this decision later, but for this merge - its bool.